src/main/fc/cli.c

```diff
@@ -1136,20 +1136,14 @@ static void cliRxRange(char *cmdline)
             int rangeMin, rangeMax;
 
             ptr = nextArg(ptr);
-            if (ptr) {
-                rangeMin = fastA2I(ptr);
-                validArgumentCount++;
-            }
+            if (!ptr) goto argumentError;
```

Comment:
I don't like usage of goto, it makes code less structured. Can we avoid using it? I.e. by having a flag that parsing is successful so far and checking for it on each block?

Reply:
I don't understand your proposed solution. I don't see how it would fix the warning. What I can do is duplicate a block `{ cliShowParseError(); return; }` for each error condition